<ui version="4.0" >
 <class>DisplayPin</class>
 <widget class="QWidget" name="DisplayPin" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>PIN method</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>230</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum" >
    <number>300</number>
   </property>
   <property name="value" >
    <number>0</number>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="format" >
    <string>Touch NFC Token in 30 sec</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPIN" >
   <property name="geometry" >
    <rect>
     <x>64</x>
     <y>192</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>00000000</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblComment" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>521</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>192</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>PIN :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
